packages/@o3r/schematics/src/rule-factories/eslint-fix/index.ts

```diff
@@ -9,29 +9,74 @@ import { getPackageManagerExecutor } from '../../utility/package-manager-runner'
 export interface LinterOptions {
   /**
    * Indicates if the linter process should succeed even if there are lint errors remaining
-   *
    * @default true
+   * @deprecated please use continueOnError, will be removed in v10
    */
-
   force?: boolean;
+  /**
+   * Indicates if the linter process should succeed even if there are lint errors remaining
+   * @default true
+   */
+  continueOnError?: boolean;
   /**
    * If enabled, only errors are reported (--quiet option of ESLint CLI)
-   *
    * @default true
    */
   hideWarnings?: boolean;
+  /**
+   * Linter configuration file path
+   */
+  configFile?: string;
+}
+
+/**
+ * Apply eslint on `files`
+ * @param files
+ * @param linterOptions
+ */
+export function eslintFix(files: string[], linterOptions: LinterOptions): Rule {
```

Comment:
This is quite confusing for 2 reasons:
- It is exposed as a rule but it is child-processing a script on the file system.
- It is using process command instead of going via the Task Api that can not make the run schedulable by the rule using it.